stock_putaway_product/wizard/quick_add_location.py

```diff
@@ -0,0 +1,79 @@
+# coding: utf-8
+# © 2017 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, models, api, fields
+
+
+class QuickAddProductLocation(models.TransientModel):
+    _name = "quick.add.product.location"
+
+    def _default_putaway(self):
+        """ Overridable method
+        """
+        return self.env.ref(
+            "stock_putaway_product.product_putaway_per_product_wh",
```

Comment:
@PierrickBrun @bealdav Missing dependency in manifest on stock_putaway_product...